### DISCUSSION TITLE
Uno.gallery github code (https://github.com/unoplatform/Uno.Gallery)

### DISCUSSION THREAD

**Discussion by modula2019** [unoplatform/uno#15484] [Q&A] [answered]
Hi support,
I tried to download the uno.gallery code (https://github.com/unoplatform/Uno.Gallery) but it seems not to compile successfully. 

GetSamples method is missing in the app.xaml.cs
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0117	'App' does not contain a definition for 'GetSamples'	Uno.Gallery.Wasm	

```
Please do you have the most recent code that you can direct me to? 
Thank

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report!

This specific repository is built under a strict CI validation and is commonly guaranteed to build properly. That being said, it may be that a specific error in a setup that prevents the source from building. Can you share a [binlog of the build](https://platform.uno/docs/articles/uno-builds-troubleshooting.html)?

**Reply by modula2019** [reply]
Hi Jerome,
I finally got it to build. Thanks for your support as always

**Reply by jeromelaban** [reply]
Thanks for the update. If you can share what you did to make it build, we may be able to enhance our documentation somehow.

**Reply by modula2019** [reply]
Hi Jerome,
I did the following to resolve it.
1. Keep it in .net 7 as it is. I was trying to upgrade it to .net8.
2. remove all d bin and obj folders
3. clean the solution
4. build d solution.
5. errors about Python not being installed may come up. If it does, follow the instruction to install python.
6. build d solution again.

The steps above should resolve it.

Thanks
